helm/cloud-provider-vsphere/templates/daemonset.yaml

```diff
@@ -100,4 +87,14 @@ spec:
       volumes:
         - name: vsphere-config-volume
           configMap:
-            name: vsphere-cloud-config
+            name: {{ if .Values.config.enabled }}{{- .Values.config.name }}{{- else }}cloud-config{{- end }}
```

Comment:
What happens if `enabled` is set to false? According to [this ](https://github.com/giantswarm/cloud-provider-vsphere-app/blob/refresh/helm/cloud-provider-vsphere/templates/configmap.yaml)there won't be any CM. In which case `cloud-config` refers to an inexistent object?